jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/java/org/jbpm/casemgmt/impl/CaseRuntimeDataServiceImplTest.java

```diff
@@ -212,6 +215,16 @@ public void testUserTasksInCase() {
         }
     }
 
+    @Test
+    public void testUserTasksInCaseAsPotentialOwners() {
+        assertTasksInCaseAsPotentialOwner("ActorId", new UserImpl("john"));
```

Comment:
As you have added the import static, it could be used here
```suggestion
        assertTasksInCaseAsPotentialOwner(ACTOR_ID, new UserImpl("john"));
```

Reply:
Done